lib/auth/accesspoint/accesspoint.go

```diff
@@ -202,5 +202,11 @@ func NewCache(cfg Config) (*cache.Cache, error) {
 		Plugin:                  cfg.Plugin,
 	}
 
-	return cache.New(cfg.Setup(cacheCfg))
+	setupCfg := cfg.Setup(cacheCfg)
+	primaryCache, err := cache.New(setupCfg)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	return primaryCache, nil
```

Comment:
This change doesn't seem necessary to me, all it does is add more lines of code to do the same thing and switches the git blame from the original author to you.

I'd revert unless there's a good reason to keep the change.